examples/k8s/simple_sat/README.md

```diff
@@ -1,6 +1,6 @@
-# Simple SPIRE 0.9.0 deployment using SAT node attestor
+# Simple SPIRE 0.10.0 deployment using SAT node attestor
 
-This configuration is an example of a simple SPIRE 0.9.0 deployment for Kubernetes that uses [SAT node attestor](https://github.com/spiffe/spire/blob/0.7.3/doc/plugin_server_nodeattestor_k8s_sat.md).
+This configuration is an example of a simple SPIRE 0.10.0 deployment for Kubernetes that uses [SAT node attestor](https://github.com/spiffe/spire/blob/0.7.3/doc/plugin_server_nodeattestor_k8s_sat.md).
```

Comment:
shouldn't we link to the latest doc for the k8s_sat server node attestor?

Reply:
done, thanks for pointing that out